reminders/plugin_bot.go

```diff
@@ -99,15 +99,19 @@ var cmds = []*commands.YAGCommand{
 		Description:         "Lists your active reminders",
 		SlashCommandEnabled: true,
 		DefaultEnabled:      true,
+		IsResponseEphemeral: true,
 		RunFunc: func(parsed *dcmd.Data) (interface{}, error) {
 			currentReminders, err := GetUserReminders(parsed.Author.ID)
 			if err != nil {
 				return nil, err
 			}
 
-			out := "Your reminders:\n"
-			out += stringReminders(currentReminders, false)
-			out += "\nRemove a reminder with `delreminder/rmreminder (id)` where id is the first number for each reminder above.\nTo clear all reminders, use `delreminder` with the `-a` switch."
+			out := "You have no reminders. Create reminders with the `Remindme` command."
```

Comment:
You use capitals here (`Remindme`) but the existing text below uses lowercase (`delreminder`.) Choose one for consistency.